index.js

```diff
@@ -2,6 +2,7 @@ const express = require('express'),
       morgan = require('morgan'),
       bodyParser = require('body-parser'),
       fs = require('fs'),
+      chalk = require('chalk'),
```

Comment:
we don't have to have chalk. I used this library to print the strings with color in the terminal